ci/get_hdf5_if_needed.sh

```diff
@@ -18,25 +18,28 @@ else
         lib_name=libhdf5.dylib
     else
         lib_name=libhdf5.so
+        ENABLE_DIRECT_VFD="--enable-direct-vfd"
```

Comment:
```suggestion
        # Test with the direct file driver on Linux. This setting does not affect the
        # HDF5 bundled in Linux wheels - that is built into a Docker image from
        # a separate repository.
        ENABLE_DIRECT_VFD="--enable-direct-vfd"
```

Just to make it clear what goes where. :slightly_smiling_face:

Reply:
Thank you!